spec/lib/fake_data/complaints_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe FakeData::Complaints do
+  describe "jsonapi_wrapper" do
+    it "should return a object with JSON-API top level JSON API" do
```

Comment:
[extreme nitpick] testing philosophies have moved away from "should" language. 

`it returns an object that matches JSON API specifications`